projects/hslayers/src/components/compositions/endpoints/compositions-layman.service.ts

```diff
@@ -88,26 +102,39 @@ export class HsCompositionsLaymanService {
     const endpoint = composition.endpoint;
     if (composition.name == undefined) {
       this.HsToastService.createToastPopupMessage(
-        'COMMON.warning',
+        this.HsLanguageService.getTranslation('COMMON.warning'),
         endpoint.title +
           ': ' +
-          'COMPOSITIONS.compostionsNameAttributeIsNotDefined',
+          this.HsLanguageService.getTranslation(
+            'COMPOSITIONS.compostionsNameAttributeIsNotDefined'
+          ),
+        true,
         'bg-warning text-light'
       );
       return;
     }
     if (endpoint.user == undefined) {
       this.HsToastService.createToastPopupMessage(
-        'COMMON.warning',
-        endpoint.title + ': ' + 'COMPOSITIONS.endpointUserIsNotDefined',
+        this.HsLanguageService.getTranslation('COMMON.warning'),
+        endpoint.title +
+          ': ' +
+          this.HsLanguageService.getTranslation(
+            'COMPOSITIONS.endpointUserIsNotDefined'
+          ),
+        true,
         'bg-warning text-light'
       );
       return;
     }
     if (endpoint.url == undefined) {
       this.HsToastService.createToastPopupMessage(
-        'COMMON.warning',
-        endpoint.title + ': ' + 'COMPOSITIONS.endpointUrlIsNotDefined',
+        this.HsLanguageService.getTranslation('COMMON.warning'),
```

Comment:
Maybe refactor issuing of common.warning inside a function, at least for this module, since its a bit repetitive.